File: modl 4/C/PRAK404-2310817210027-MuhammadErzaRaihan.c
#include <stdio.h>
#include <stdlib.h>

void menu(){
    printf("Pilih Program\n");
    printf("1. Penjumlahan\n");
    printf("2. Pengurangan\n");
    printf("3. Perkalian\n");
    printf("4. Pembagian\n");
    printf("5. Exit\n");
}

void exit_message(){
    printf("Terimakasih, telah menggunakan kalkulator Muhammad Erza Raihan\n");
}

int main(){
    int operasi;
    float A, B, hasil;
    while(1){
        menu();
        printf("Masukkan Pilihan : ");
        scanf("%d", &operasi);
        if(operasi == 5){
            exit_message();
            break;
        }
        if(operasi > 5){
            printf("Input anda salah, silahkan coba lagi\n");
            continue;
        }
        printf("Masukkan bilangan pertama: ");
        scanf("%f", &A);
        printf("Masukkan bilangan kedua: ");
        scanf("%f", &B);

        if(operasi == 1){
            hasil = A + B;
            printf("Hasil Penjumlahan antara %.2f dengan %.2f adalah %.2f\n", A, B, hasil);
        }else if(operasi == 2){
            hasil = A - B;
            printf("Hasil Pengurangan antara %.2f dengan %.2f adalah %.2f\n", A, B, hasil);
        }else if(operasi == 3){
            hasil = A * B;
            printf("Hasil Perkalian antara %.2f dengan %.2f adalah %.2f\n", A, B, hasil);
        }else if(operasi == 4){
            hasil = A / B;
            printf("Hasil Pembagian antara %.2f dengan %.2f adalah %.2f\n", A, B, hasil);
        }
    }
    return 0;
}

File: modl 4/python/PRAK404-2310817210027-MuhammadErzaRaihan.py
print('Pilih Program')
print('1. Penjumlahan')
print('2. Pengurangan')
print('3. Perkalian')
print('4. Pembagian')
print('5. Exit')

while True:
    op = input('Masukkan Pilihan :')

    if op == '5':
        print('Terimakasih, telah menggunakan kalkulator Muhammad Erza Raihan')
        break
    if op not in ['1','2','3','4','5']:
        print('Input anda salah, silahkan coba lagi')
        continue
    
         

    A = float(input('Masukkan bilangan pertama: '))
    B = float(input('Masukkan bilangan kedua: '))

    if op == '1':
        hasil = A + B
        print(f'Hasil Penjumlahan antara {A:.2f} dengan {B:.2f} adalah {hasil:.2f}')
    elif op == '2':
        hasil = A - B
        print(f'Hasil Pengurangan antara {A:.2f} dengan {B:.2f} adalah {hasil:.2f}')
    elif op == '3':
        hasil = A * B
        print(f'Hasil Perkalian antara {A:.2f} dengan {B:.2f} adalah {hasil:.2f}')
    elif op == '4':
          hasil = A / B
          print(f'Hasil Pembagian antara {A:.2f} dengan {B:.2f} adalah {hasil:.2f}')
   